CHANGELOG.md

```diff
@@ -45,6 +45,10 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Changing the (Kind) drop-down according to (Category) drop-down in Managed variant add variant
 - Moved Gens button to individuals table
 
+## [4.37]
```

Comment:
We don't deploy every time we merge to master, see branching model documentation: https://github.com/Clinical-Genomics/scout/blob/master/CONTRIBUTING.md (sorry if it's not super clear!). You should branch out from current master (or pull current master into this branch) and add this features in the adds of the next release (still tagged as x.x.x). It has a long list of features